src/aspire/source/image.py

```diff
@@ -567,6 +567,8 @@ def save(
         :param batch_size: Batch size of images to query.
         :param save_mode: Whether to save all images in a single or multiple files in batch size.
         :param overwrite: Option to overwrite the output MRCS files.
+        :return: A dictionary containing "starfile"--the path to the saved starfile-- and "mrcs", a
+        list of the saved particle stack .mrcs filenames.
```

Comment:
Hey, I didn't realize this was happening before Josh pointed it out to me, but I think the line beginning with `list` needs to be indented for Sphinx to parse it.

Reply:
Cool, had it checked out and rebuilt the doc, lgtm. Thanks!